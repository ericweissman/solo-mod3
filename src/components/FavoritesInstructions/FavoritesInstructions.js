import React from 'react'

const FavoritesInstructions = () => {
  return (
    <div className='favoritesInstructions'>
      <h4>No Favorites Yet!</h4>
      <p className='instruction'>To add a favorite, click on one of the music genres on the left and explore some of the artists. If you like a particlar aritst, click the Favorite button and their info will be saved here for your next visit!</p>
    </div>
  )
}

export default FavoritesInstructions