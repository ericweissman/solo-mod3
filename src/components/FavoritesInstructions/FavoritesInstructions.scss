.favoritesInstructions {
  @extend .instructions;
  h4 {
    font-size: 35px;
  }
  p {
    @extend .card;
    height: 175px;
    line-height: 25px;
  }
}