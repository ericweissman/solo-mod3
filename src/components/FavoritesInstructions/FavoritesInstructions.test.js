import React from 'react'
import FavoritesInstructions from './FavoritesInstructions'
import { shallow } from 'enzyme'

describe('FavoritesInstructions', () => {
  let wrapper
  it('should match the correct snapshot', () => {
    wrapper = shallow(<FavoritesInstructions />)
    expect(wrapper).toMatchSnapshot()
  })
})