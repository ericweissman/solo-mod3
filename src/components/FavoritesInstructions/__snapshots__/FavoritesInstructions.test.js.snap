// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesInstructions should match the correct snapshot 1`] = `
<div
  className="favoritesInstructions"
>
  <h4>
    No Favorites Yet!
  </h4>
  <p>
    To add a favorite, click on one of the music genres on the left and explore some of the artists. If you like a particlar aritst, click the Favorite button and their info will be saved here for your next visit!
  </p>
</div>
`;
